### PR TITLE
enchannel 1.1.0 spec

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,6 +11,26 @@ import {
...
+ * @param  {string} identity                UUID
+ * @param  {Object} config                  Jupyter connection information
+ * @param  {string} config.ip               IP address of the kernel
+ * @param  {string} config.transport        Transport, e.g. TCP
+ * @param  {string} config.signature_scheme Hashing scheme, e.g. hmac-sha256
+ * @param  {number} config.iopub_port       Port for iopub channel
+ * @param  {string} subscription            subscribed topic; defaults to all
+ * @return {object} channels object, per enchannel spec
+ */
+export function createChannels(identity, config, subscription='') {
```
Comment (rgbkrk) [inline src/index.js:24]: The IOPub subscription is so weird here, makes sense though.

File: README.md [hunk 2]
```diff
@@ -46,18 +48,13 @@ Provides functions to create [RxJS](https://github.com/ReactiveX/RxJS) subjects
 
 ## Usage
 
-`enchannel-zmq-backend` exports four functions to create channels
+`enchannel-zmq-backend` exports a `createChannels` method for creating a channels object per [enchannel spec](https://github.com/nteract/enchannel).
```
Comment (rgbkrk) [inline README.md:51]: I'd love to re-phrase this as:

> To get access to all of the `channels` for messaging, use `createChannels`:

After Travis passes and I merge I'll follow up.

**Comment by jdfreder** [comment]
Good to go!

File: README.md (hunk 2 above)
Comment (jdfreder) [inline README.md:51]: Okay, I'll leave that to you

**Pull request merged**